specification/eventgrid/data-plane/Microsoft.ApiManagement/stable/2018-01-01/APIManagement.json

```diff
@@ -0,0 +1,161 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure API Management events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure API Management events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "UserCreatedEventData": {
```

Comment:
We use the names of these `EventData` definitions for the corresponding model types in each language.  Some of these names are fairly general, and could be similar to say events from other sources (for example, I think ARM publishes similar events about subscriptions).

For the ones that don't already have an Api prefix, could we add something like `ApiManagement`, e.g. `ApiManagementUserCreatedEventData` and `ApiManagementSubscriptionCreatedEventData`